open-metadata-resources/open-metadata-deployment/charts/vdc/values.yaml

```diff
@@ -46,42 +47,21 @@ service:
     externalip: 10.10.1.30
     # KAFKA is not exposed externally (issues in chart, complexity)
 
-# IGC configuration
-igc:
-  host: 158.176.66.119
-  hostname: igc.openmetadata.ibmcloud.com
-  webport: 9446
-  kafkaport: 59092
-  auth:
-    os:
-      user: ibm
-      pass: letme1n2
-    app:
-      user: isadmin
-      pass: isadmin
-
 # Debug support for Egeria - set to true to enable JVM debug for Egeria images on port 5005
 debug:
-    egeriaJVM: false
+  egeriaJVM: false
 
 # Logging level - affects all spring components by setting LOG_LEVEL_ROOT. Default to no DEBUG - Auditlog only
 logging:
-    level: NONE
+  level: NONE
 
 
 # The following section defines all of the DOCKER images being used by this
 # chart. Normally they should be left as is, but are exposed here in case the
 # user wishes to extend.
-
-# Images ending -init are typically containers that perform configuration,
-# often in the same container as the component they are configuring
-
-# Note that sometimes the same docker image is used for multiple purposes - for example
-# Both the Egeria UI, and Egeria core use the same docker image -- for now
-
 image:
   ranger:
-    repository: planetf1/ranger-admin-egeriavdc
```

Comment:
Neither of these are perfect. at least planetf1 is externally resolveable, though it *should* be ODPi. but it can be yet Suggestions? I think if nothing better we should revert to the original as at least anyone can then take the chart. I will chase up build team to get an ODPi account on docker hub.

Reply:
The problem is that with that approach no one else can *build* (and push / re-use) the charts (ie. updated ones)...

Reply:
I know :-) I was focussed on the consumer in the first case ie anyone can install our charts.
I've opened up https://github.com/odpi/ci-management/issues/44 to address this with the ODPi/LF team so let's chase that.